### ISSUE TITLE
BUG: score_by_content() fails — get_content_activations() returns tensor, not iterable

### ISSUE THREAD

**Issue by yuenhy** [yuenhy/stapler#4]
## Bug Report

**Severity:** High — runtime error when using content scoring

**Files:** `distance/vgg.py` line 20–21, `distance/neural_style.py` line 92

**Description:**
`get_content_activations()` returns a single tensor, but `score_by_content()` calls `[x.detach() for x in generated_activations]` which attempts to iterate over it as if it were a list/tuple of tensors. This will raise a dimension or unpacking error.

**Affected Code:**
```python
# vgg.py
def get_content_activations(self, x):
    features = self.vgg[:23](x)
    return features  # single tensor

# neural_style.py
generated_gram = [x.detach() for x in generated_activations]  # iterates as list
```

**Fix:**
Either wrap the return in a list `return [features]`, or update `neural_style.py` to handle the tensor directly without list comprehension.
